packages/client/base/src/services/embed/v3/crossmintEmbeddedCheckoutV3Service.ts

```diff
@@ -4,9 +4,15 @@ import { embeddedCheckoutV3IncomingEvents, embeddedCheckoutV3OutgoingEvents } fr
 import { IFrameWindow } from "@crossmint/client-sdk-window";
 import type { CrossmintApiClient } from "@crossmint/common-sdk-base";
 
-export type CrossmintEmbeddedCheckoutV3ServiceProps = { apiClient: CrossmintApiClient };
+export type CrossmintEmbeddedCheckoutV3ServiceProps = { 
+    apiClient: CrossmintApiClient,
+    sdkMetadata: {
```

Comment:
i think we have access to `CrossmintApiClientInternalConfig["sdkMetadata"]` from here, better to keep in sync